### PR TITLE
[receiver/mongodbatlas] fix attribute key on disk partition metrics

### PR DESCRIPTION
**Pull request by mstumpfx**
**Description:** These should have all been using the `disk_status` key for the attributes, not `memory_state`.

### CODE REVIEW

**Review by zenmoto — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

File: receiver/mongodbatlasreceiver/metadata.yaml [hunk 1]
```diff
@@ -579,15 +579,15 @@ metrics:
     description: Disk partition space
     extended_documentation: Aggregate of MongoDB Metrics DISK_PARTITION_SPACE_FREE, DISK_PARTITION_SPACE_USED
     unit: By
-    attributes: [memory_state]
+    attributes: [disk_status]
```
Comment (bogdandrutu) [inline receiver/mongodbatlasreceiver/metadata.yaml:582]: Why do we need to repeat "disk"? "status" is not enough?

As example for host metrics we don't have "memory_state" and "cpu_state".

File: receiver/mongodbatlasreceiver/receiver.go [hunk 2]
```diff
@@ -53,9 +53,9 @@ func newMongoDBAtlasScraper(log *zap.Logger, cfg *Config) (scraperhelper.Scraper
 func (s *receiver) scrape(ctx context.Context) (pdata.Metrics, error) {
 	var start time.Time
 	if s.lastRun.IsZero() {
-		start = s.lastRun
-	} else {
 		start = time.Now().Add(s.cfg.CollectionInterval * -1)
+	} else {
```
Comment (codeboten) [inline receiver/mongodbatlasreceiver/receiver.go:57]: This change is unrelated to the one described in the PR. Should there be a separate line item in the changelog for it?

Comment (mstumpfx) [reply receiver/mongodbatlasreceiver/receiver.go]: Whoops, got my branches mixed up, thanks 🙂
